shell/browser/ui/cocoa/electron_menu_controller.mm

```diff
@@ -315,12 +315,21 @@ - (NSMenuItem*)menuItemForService:(NSSharingService*)service
 - (NSMenuItem*)makeMenuItemForIndex:(NSInteger)index
                           fromModel:(electron::ElectronMenuModel*)model {
   std::u16string label16 = model->GetLabelAt(index);
+  auto rawSecondaryLabel = model->GetSecondaryLabelAt(index);
   NSString* label = l10n_util::FixUpWindowsStyleLabel(label16);
 
   NSMenuItem* item = [[NSMenuItem alloc] initWithTitle:label
                                                 action:@selector(itemSelected:)
                                          keyEquivalent:@""];
 
+  if (!rawSecondaryLabel.empty()) {
+    NSString* secondary_label =
+        l10n_util::FixUpWindowsStyleLabel(rawSecondaryLabel);
+    if (@available(macOS 14.4, *)) {
+      item.subtitle = secondary_label;
+    }
```

Comment:
Minor nit to avoid l10n when not needed

```suggestion
     if (@available(macOS 14.4, *)) {
      NSString* secondary_label =
          l10n_util::FixUpWindowsStyleLabel(rawSecondaryLabel);
      item.subtitle = secondary_label;
    }
```